### PR TITLE
guri46's vancouver food list

### PR DESCRIPTION
**Pull request by codegemini**
Added places to eat.

@beardofedu : can you check this out?

### CODE REVIEW

File: guri46-vancouver.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Vancouver, BC, CA
+
+### Places to eat
+- Meet on Main for vegetarian food
+- Nuba for Lebanese food
+- Tap & Barrel for everything!!
```
Comment (JennieOhyoung) [inline guri46-vancouver.md]: Do they serve brunch?

File: guri46-vancouver.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## Vancouver, BC, CA
+
+### Places to eat
```
Comment (JennieOhyoung) [inline guri46-vancouver.md:3]: Awesome 😋

**Review by JennieOhyoung — APPROVED** [review]
👍

File: guri46-vancouver.md (hunk 1 above)
Comment (codegemini) [reply guri46-vancouver.md]: Yes, Tap & Barrel serves beer instead of Coffee in mornings ;)

**Pull request merged**